paper/mitchell-geophysics.tex

```diff
@@ -0,0 +1,1999 @@
+
+\subsection{Offset TX Electrode Trials}
+\label{sec:TheoreticalAnalysis_OffsetTrials}
+
+Using the equation derived by \citet{Wait1982m} the analytic electrical potential due to a sphere excited by a current point can be computed at any RX location outside of the sphere. A summary of this derivation can also be found in \citet{Mitchell2020}.  Starting with a linear array of electrodes extending in the positive $x$-direction we calculated the pole-pole response of 4 spheres which are centered at (25, 8, 0 m), (25, 0, 8 m), (25, -8, 0 m), and (25, 0, -8 m), respectively, to a TX located at (25, 0, 0 m). The radius of the spheres was chosen so that they would have the same volume as a 5 m cube. A background conductivity of $5 \times 10^{\text{-3}}$ S/m and a sphere conductivity of 10 S/m were used in keeping with values from the potash salt mine environment. The geometry of this experiment is shown in Figure \ref{fig:4Spheres_ObliqueView} and the resulting potentials are shown in Figure \ref{fig:4Spheres_Symmetric}. With each trial that follows, one or more offset TX electrodes are added to the array and the differences in response are evaluated. Figure \ref{fig:4Spheres_TX_ZOffset} shows an example with a single TX electrode offset in the positive $z$-direction.
+
+\begin{figure}[htp]
+   \begin{center}
+      \includegraphics[trim=0cm 0.0cm 0cm 0.0cm, clip=true,width=0.65\linewidth]{./figures/Fig1.png}
+   \end{center}
+\caption{Oblique view of the experimental setup for a linear array of electrodes embedded in a fullspace. The analytic response of a pole-pole survey with the TX electrode at (25, 0, 0 m) is calculated for 4 separate conductive spheres centered at: (25, 8, 0 m), (25, 0, 8 m), (25, -8, 0 m), and (25, 0, -8 m), respectively. In this Figure Sphere \#1 is solid while the other spheres are translucent.}
+\label{fig:4Spheres_ObliqueView}
+\end{figure}
+
+To assess the performance of each survey the problem was discretized on an octree mesh with 1 m cubic cells in the core region and a 5 m conductive cube was placed at the same center location as sphere 1 (25, 8, 0 m). Synthetic data were then forward modeled and inverted for each survey. To minimize complicating factors and promote the best possible recovery of the conductive block no noise was added to the forward modeled data and uncertainties were set using only a small floor value of $1 \times 10^{\text{-6}}$ V/A.
+
+
+\subsubsection{Trial 1: Single Linear Array}
+\label{sec:TheoreticalAnalysis_Trial1_SingleLinearArray}
+
+Due to the symmetry of the single linear array, any sphere of the same radius and conductivity that is centered on the dotted circle in Figure \ref{fig:4Spheres_Symmetric_Model} will produce an identical secondary response, as shown in panel b). Since there is no difference in the modeled data, the inversion has no way to constrain the true location of the sphere.
+
+
+\begin{figure}[htp]
+   \begin{center}
+      \begin{subfigure}{0.4\linewidth}

```

Comment:
And one more question, what would you recommend for subfigures? It doesn't look like myst catches these at the moment?